convertio/client.py

```diff
@@ -64,7 +64,8 @@ def convert_by_filename(self,
 
         r = request.Request(
             url='https://api.convertio.co/convert',
-            data=json.dumps(data).encode("utf8"),
+            data=json.dumps({k: base64.b64encode(v).decode('utf-8') if isinstance(v, bytes) else v for k, v in

```

Comment:
You are forced to decode here what you encoded line 50. You can instead decode it there, like
```python
 'file': base64.b64encode(file.read()).decode()
```